util/openocd/interface/bergen-mpsse.cfg

```diff
@@ -0,0 +1,28 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Interface configuration for JTAG on the CW310 Board via on-board
+
+# adapter speed
+# Doesn't matter what you say here, hardware ignores it
+adapter_khz 500
```

Comment:
When I run it I got this warning `DEPRECATED! use 'adapter speed' not 'adapter_khz`. Isn`t it worth changing?

Reply:
I can update that one. I followed the same format as `nexysvideo-ft2232.cfg` uses in the current file (I think the ftdi config parameters give the same warning?). I wasn't sure if we still needed to support the "old" format for some reason.

Reply:
I don't have a strong opinion on that, so let's wait for someone else to review.

Additionally the CW310 board doesn't have a FTDI chip embedded right? Should I use a cable instead?

Reply:
The CW310 doesn't have a FTDI chip, but instead it emulates the FTDI MPSEE interface (aka the JTAG interface code). This lets you talk to the CW310 "as if" it was an FTDI chip attached as well. You need to enable this mode since it disables some of the USB-CDC serial interfaces as they use the same USB resources. The `ftdi_vid_pid 0x2b3e 0xc310` line tells OpenOCD to talk to the CW310 USB endpoints "as if" it was an FTDI, basically this was done as it means you don't need a special OpenOCD build to support the CW310 (in our experience people end up with various builds/versions of OpenOCD due to other compatibility issues... so we really wanted to force needing to use our own fork of OpenOCD).

Reply:
I agree with @doreis-lowrisc, I would also prefer no warnings.